#language: ru

@tree

Функционал: применение попытки исключения

Как <Роль> я хочу
<описание функционала> 
чтобы <бизнес-эффект> 

Контекст:
	Дано Я запускаю сценарий открытия TestClient или подключаю уже существующий
	// необходимо вручную открыть внешнюю обработку 
	//D:\Cloud\Dev\GitRep\TestQA\Lessons\Features\Internal\Конструкции языка Turbo Gherkin\ПроверкаЗначенияГода.epf

Сценарий: тест веб сервиса
	И в поле с именем 'ПроверитьГод' я ввожу текст '2020'
	И Я запоминаю в переменную "СоединениеУстановлено" значение "Нет"
	И я делаю 2 раз
		Попытка
			И я нажимаю на кнопку с именем 'ФормаПолучитьДанные'
			И элемент формы с именем "ГодВысокосный" стал равен 'Да'
			И Я запоминаю в переменную "СоединениеУстановлено" значение "Да"
			Тогда я прерываю цикл
		Исключение
			И Пауза 3
	Если переменная "СоединениеУстановлено" имеет значение "Нет" Тогда
		Затем я останавливаю выполнение сценария "Skipped"
	Дано Я открываю навигационную ссылку "e1cib/list/Catalog.Stores"
		
Сценарий: тест веб сервиса
	Попытка
		И Я подключаю клиент тестирования "ИмяКлиентаТестирования" из таблицы клиентов тестирования
	Исключение
		И я регистрирую ошибку "Первая ошибка" по данным исключения
	И Пауза 2
	Попытка
		И Я подключаю клиент тестирования "ИмяКлиентаТестирования" из таблицы клиентов тестирования
	Исключение
		И я регистрирую ошибку "Вторая ошибка" по данным исключения						
		

		
	